RyuSocks/Packets/UaPPacket.cs

```diff
@@ -0,0 +1,21 @@
+namespace RyuSocks.Packets
+{
+    public struct UaPPacket(string username, string password) : IAuthenticationPacket
+    {
+        public byte Version { get; set; }
+        public int UsernameLength = username.Length;
+        public string Username = username;
+        public int PasswordLength = password.Length;
+        public string Password = password;
```

Comment:
Don't set default values here and instead move all of that to the constructor.

Reply:
no default values set now